assests/styleSheets/main.css

```diff
@@ -0,0 +1,81 @@
+@import "./reset.css";
+@import "./variables.css";
+
+body {
+  padding: var(--space-md);
+  height: max-content;
```

Comment:
I think min-height: 96vh works better here.

Reply:
Thanks ,I changed "height" to "min-height" but the number which worked correctly was 92.5.